src/code/utilities/url-params.js

```diff
@@ -12,4 +12,16 @@ var urlParams;
        urlParams[decode(match[1])] = decode(match[2]);
 })();
 
+export function getClassId() {
+  // Pull the class number out of the class info
+  let regex = /classes\/(\d*)/g,
+      matches = regex.exec(urlParams.class_info_url);
+
+  return matches ? matches[1] : null;
+}
+
+export function getUserId() {
+  return urlParams.domain_uid;
```

Comment:
The domain_uid is named that way because this is a generic feature that used for single sign on. Where the 'domain' is the SSO provider.  As I wrote below I'd suggest combining the domain and the domain_uid, so it will work if GV is used in multiple portals.